runtime/doc/lsp.txt

```diff
@@ -2380,12 +2380,14 @@ stylize_markdown({bufnr}, {contents}, {opts})
     Return: ~
         (`table`) stripped content
 
-symbols_to_items({symbols}, {bufnr})         *vim.lsp.util.symbols_to_items()*
+                                             *vim.lsp.util.symbols_to_items()*
+symbols_to_items({symbols}, {bufnr}, {position_encoding})
```

Comment:
Why did the order change here? I think `*vim.lsp.util.symbols_to_items()*` should still be aligned to the right.